votes/initiateNewVote/_EDIT_ME.yml

```diff
@@ -35,7 +36,10 @@ canShuffleCandidates: true
 # 5. Insert here a short description of the vote objectives and link to the
 #    issue it was discussed on to give the full context.
 footerInstructions: |
-  TBD
+  The objective of the vote is to determine a basic question: do we continue bundling
+  the corepack binary with Node.js or not, and if so, do we enable bundling jumper
+  binaries for its supported package managers (such as yarn and pnpm) by default or not.
+  There are additional questions to resolve that will be handled separately.
 
 # 6. Optionally, insert a brief introduction for the vote PR, in the markdown format.
 prBody: |
```

Comment:
```suggestion
prBody: |
  There are several different visions being worked out about the future distribution
  strategy of package managers and Node.js itself. Some of them involve a bundled
  corepack, some of them need a standalone corepack, some of them do not involve
  corepack. We still have not decided on which vision we want. This vote is
  specifically about whether we should continue bundling corepack before we
  decide on the vision of our distribution strategy. It is separate from whether
  corepack should eventually stay in the organization or be part of the
  future of Node.js, as those are still unclear until we decide about our vision
  in the distribution strategy.
```

Reply:
Alternatively we can just decide what vision we want, and presumably based on that this vote won’t be necessary.